content/decadal/2_hindcasts.md

```diff
@@ -0,0 +1,57 @@
+---
+tag: Hindcasts
+title: Decadal hindcasts in EUCP
+img: decadal_hindcast.png
+caption: Timeseries for Global Mean Near Surface Temperature
```

Comment:
Please include the source.

Reply:
The source link is added to the caption.